content/docs/user-documentation/configuring.md

```diff
@@ -10,7 +10,69 @@ summary: Configuration options with MicroShift
 modified: "2021-11-03T16:24:11.538+01:00"
 ---
 
-## WIP Content coming soon
+<!-- 
+todo: 
+- logging 
+-->
 
-- config.yaml vs env vars vs command line
-- logging
+## Configuration
+
+Microshift can be configured in three simple ways, in order of precedence: 
+- Commandline arguments
+- Environment variables
+- Configuration file
+
+Values are passed in to modify the `MicroshiftConfig` defined in `pkg/config/config.go`. 
```

Comment:
As this is a User Documentation, I wouldn't mention or refer to any code details at all. For example, our code may change or introduce experimental flags (that we'd hide from CLI) and that shouldn't be visible to users.

Reply:
That makes sense